dts/bindings/gpio/nxp,kinetis-gpio.yaml

```diff
@@ -18,6 +18,17 @@ properties:
       A phandle reference to the device tree node that contains the pinmux
       port associated with this GPIO controller.
 
+  int-source:
+    type: string
+    default: "none"
+    enum
+      - "int-a"
+      - "int-b"
+      - "int-c"
+      - "none"
+      description: |
+        Interrupt source for the gpio port
```

Comment:
needs better description, also, is this something soc specific

Reply:
The number of int-X is soc specific but the enum is not SOC specific, this is seen on other SOC parts via lpc-gpio

Reply:
how is lpc gpio related to this PR, I dont understand what you mean

Reply:
I don't think these gpios have configurable interrupt sources, so this property should be removed